services/src/main/scala/cromwell/services/metadata/impl/archiver/DeleteMetadataActor.scala

```diff
@@ -2,10 +2,17 @@ package cromwell.services.metadata.impl.archiver
 
 import akka.actor.{Actor, ActorLogging, ActorRef}
 import com.typesafe.config.Config
+import common.util.TimeUtil.EnhancedOffsetDateTime
 import cromwell.services.MetadataServicesStore
 import cromwell.services.instrumentation.CromwellInstrumentation
 import cromwell.services.metadata.impl.MetadataDatabaseAccess
 
+import scala.concurrent.duration.FiniteDuration
+
+/*
+  This class would do same things cromwell.services.metadata.hybridcarbonite.DeleteMetadataActor does
+ */
+
```

Comment:
This comment can be removed as `hybridcarbonite.DeleteMetadataActor` does not exist any longer

Reply:
👍

Reply:
Actually, since this is entirely unused here _and_ replaced in https://github.com/broadinstitute/cromwell/pull/6232, I think I'll just roll back all changes in this file for now and let that PR implement the deleter